open-source-terms.md

```diff
@@ -0,0 +1,259 @@
+# npm Open-Source Terms
+
+These npm Open Source terms of use (these _Terms_) govern access to
+and use of <https://www.npmjs.com> (the _Website_) as well as the
+"npm Public Registry" at <https://registry.npmjs.com> (the _Public
+Registry_). Both the Website and the Public Registry are operated by
+npm, Inc. (_npm_). The Website and the Public Registry are referred to
+together as _npm Open Source_ throughout these Terms.
+
+These npm Open Source Terms were last updated on November 9, 2015.
+
+## Important Terms
+
+***These Terms include a number of important provisions that affect your
+rights and responsibilities, such as the disclaimers in "Disclaimers",
+limits on npm's liability to you in "Limits on Liability", and an
+agreement to arbitrate disputes individually in "Arbitration".***
+
+## Other Terms
+
+npm offers additional services (_Paid Services_) that are subject to
+additional terms. Additional terms for personal private packages are
+found at <https://www.npmjs.com/policies/personal-terms>.
```

Comment:
Yes. This PR already includes "add-on" terms for Personal. The same "modular" approach would work for Orgs terms with minimal changes.